test/web-platform-tests/to-upstream/html/webappapis/scripting/processing-model-2/window-onerror-event-throw-null.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>window.onerror: throwing null in event listeners</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  "use strict";
+
+  setup({ allow_uncaught_exception: true });
+
+  test(() => {
+    let errorEvents = 0;
+    window.addEventListener("error", event => {
+      assert_equals(event.message, null);
```

Comment:
It looks like we technically cannot use `null` as `event.message`, as it is [defined](https://html.spec.whatwg.org/multipage/webappapis.html#runtime-script-errors:dom-errorevent-message-2) to always be a string.

I suggest this logic, which emulates the Firefox behavior:
```js
let message;
if (error === undefined || error === null || typeof error === "boolean" ||
    typeof error === "number" || typeof error === "bigint" || typeof error === "symbol") {
  message = "uncaught exception: " + String(error);
} else if (error.message) {
  message = String(error.message);
} else {
  message = "uncaught exception";
}
```

Reply:
Updated. The code I added matches the code for unhandled exceptions (line 65).